cli/command/image/build.go

```diff
@@ -246,7 +246,11 @@ func runBuild(dockerCli command.Cli, options buildOptions) error {
 	)
 
 	if options.stream {
-		return errors.New("Experimental flag --stream was removed, enable BuildKit instead with DOCKER_BUILDKIT=1")
+		_, _ = fmt.Fprint(dockerCli.Err(), `DEPRECATED: The experimental --stream flag is deprecated and the build context
+            will be sent non-streaming. Enable BuildKit instead with DOCKER_BUILDKIT=1
```

Comment:
"Consider using BuildKit instead", maybe with a doc link?

Reply:
Good suggestion; I recently added https://github.com/docker/docker.github.io/pull/11460, and this would be a good candidate for that. Need to check if we have a good location already to point users to (on how to enable buildkit).

can then add a URL like `https://docs.docker.com/go/buildkit`

Reply:
Opened https://github.com/docker/docker.github.io/pull/11621 to implement `https://docs.docker.com/go/buildkit/` as a redirect